src/cromshell/utilities/CromshellConfig.py

```diff
@@ -0,0 +1,127 @@
+                        column = line.split()
+                if column:
+                    cls.LOGGER.info("Found workflow id in submission file.")
+                    cls.cromwell_server = column[1]
+                    CromshellConfig.LOGGER.info(
+                        "Cromwell server set to matching workflow id in submission file."
+                    )
+                    CromshellConfig.LOGGER.info(cls.cromwell_server)
+                else:
+                    cls.LOGGER.info(
+                        "Unable to find workflow id in submission file. Using default"
+                    )
+
+    @classmethod
+    def user_defined_show_logo(cls, verbosity):
+        """Boolean for Displaying Turtle Logo"""
+
+        if verbosity is None or verbosity < logging.CRITICAL:
+            cls.is_verbose = True
+        else:
+            cls.is_verbose = False
+
+    @staticmethod
+    def __get_config_dir():
+        """Get Path To Cromshell Hidden Directory"""
+
+        config_path = str(Path.home()) + "/.cromshell"
+        Path.mkdir(Path(config_path), exist_ok=True)
+        return config_path
+
+    @staticmethod
+    def __get_submission_file(config_dir):
+        """Get File Path To Cromshell Submission File"""
+
+        submission_file = "{}/all.workflow.database.tsv".format(config_dir)
+        if not Path.exists(Path(submission_file)):
+            Path.touch(Path(submission_file))
+            submission_header = (
+                "DATE{0}CROMWELL_SERVER{0}RUN_ID{0}WDL_NAME{0}STATUS{0}ALIAS".format(
+                    "\t"
+                )
+            )
+            with Path(submission_file).open() as f:
+                f.write(submission_header)
+        return submission_file
+
+    @staticmethod
+    def __get_cromwell_server(config_dir):
+        """Get Cromshell Server URL from Configuration File"""
+
+        cromwell_server_path = "{}/cromwell_server.config".format(config_dir)
+        if Path.exists(Path(cromwell_server_path)):
+            with Path(cromwell_server_path).open() as f:
+                cromwell_server = f.readline().strip()
+            return cromwell_server
+            CromshellConfig.LOGGER.info(
+                "Setting cromwell server to cromwell url from config file."
+            )
+            CromshellConfig.LOGGER.info(cromwell_server)
+        else:
+            CromshellConfig.LOGGER.error("Could not find cromwell_server.config")
+            CromshellConfig.LOGGER.info("Creating file %s", cromwell_server_path)
+            Path.touch(Path(cromwell_server_path))
+            CromshellConfig.LOGGER.error(
+                "Please add a Cromwell server URL to file %s", cromwell_server_path
+            )
+            sys.exit()
+
+    LOGGER = logging.getLogger(__name__)
+
+    # Set Cromshell Configuration Default Values
+    metadata_parameters = "excludeKey=submittedFiles&expandSubWorkflows=true"
+    config_dir = __get_config_dir.__func__()
```

Comment:
You shouldn't need the `__func__` here.  You may need to call them with the full class specifier though (`CromshellConfig .__get_submission_file(config_dir)`).